dashboards-reports/server/routes/utils/constants.ts

```diff
@@ -167,6 +169,18 @@ export const GLOBAL_BASIC_COUNTER: CountersType = {
       },
     },
   },
+  notebook: {
+    pdf: {
+      download: {
+        count: 0,
```

Comment:
are notebooks metrics recorded or just a place holder for now?

Reply:
This is a placeholder for now, notebooks metrics will need to be added